.github/workflows/validate-lavamoat-policy-webapp.yml

```diff
@@ -0,0 +1,30 @@
+name: Validate lavamoat policy webapp
+
+on:
+  workflow_call:
+
+jobs:
+  validate-lavamoat-policy-webapp:
+    name: Validate lavamoat policy webapp
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build-type: [main, beta, flask, mmi]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: metamask/github-tools/.github/actions/setup-environment@main
+
+      - name: Validate lavamoat ${{ matrix.build-type }} policy
+        run: yarn lavamoat:webapp:auto:ci --build-types=${{ matrix.build-type }}
+        env:
+          INFURA_PROJECT_ID: 00000000000
```

Comment:
Good question! This is the single required environment variable for our build process. Generating an application LavaMoat policy requires building the project, so we needed to set it to _something_. But since this is a mock value that we don't want to use elsewhere, it's set inline directly.

The same strategy is used in the existing "Update LavaMoat policies" workflow: https://github.com/MetaMask/metamask-extension/blob/695d0db025fff9d9b29d6ca2c03c42bfd58cf57e/.github/workflows/update-lavamoat-policies.yml#L120

Reply:
Ah! Thanks for the explanation!